--- a/osdep/LinuxEthernetTap.cpp	2022-06-28 05:13:57.000000000 +0800
+++ b/osdep/LinuxEthernetTap.cpp	2022-10-31 14:15:42.626923438 +0800
@@ -275,11 +275,11 @@
 		if (!isOldLinuxKernel()) {
 			ifr.ifr_ifru.ifru_hwaddr.sa_family = ARPHRD_ETHER;
 			_mac.copyTo(ifr.ifr_ifru.ifru_hwaddr.sa_data,6);
-			if (ioctl(sock,SIOCSIFHWADDR,(void *)&ifr) < 0) {
-				::close(sock);
-				printf("WARNING: ioctl() failed setting up Linux tap device (set MAC)\n");
-				return;
-			}
+			// if (ioctl(sock,SIOCSIFHWADDR,(void *)&ifr) < 0) {
+			// 	::close(sock);
+			// 	printf("WARNING: ioctl() failed setting up Linux tap device (set MAC)\n");
+			// 	return;
+			// }
 
 			ifr.ifr_ifru.ifru_mtu = (int)_mtu;
 			if (ioctl(sock,SIOCSIFMTU,(void *)&ifr) < 0) {
